sys-kernel/coreos-sources/coreos-sources-4.12.8.ebuild

```diff
@@ -3,18 +3,29 @@
 
 EAPI="5"
 ETYPE="sources"
+
+# -rc releases should be versioned L.M_rcN
+# Final releases should be versioned L.M.N, even for n == 0
+
+# Only needed for RCs
+K_BASE_VER="4.12"
```

Comment:
If this is only for RCs (as `kernel-2.eclass` seems to indicate), can the value just be `${PV%%_*}` so we don't have to edit this file for updates?

Reply:
Ah, nevermind, I see that this same 4.12 version is carried over for 4.13 for the tar version.